dot_default-npm-packages

```diff
@@ -0,0 +1,3 @@
+typescript-language-server
```

Comment:
This file is used by asdf when install a new version of node - this will be useful when jumping between projects using different version of node - not sure how necessary it is though as which version of node is neovim using? The on in the repo it was opened in or my user level one 🤷‍♂️